src/client/components/DeliveryInfo/DeliveryInfo.styles.css

```diff
@@ -0,0 +1,119 @@
+form {
```

Comment:
Double check your css with the mockup, to follow the correct styles

Reply:
Hi, thank you for the review. Is it possible to get the html from the mockups? To align the correct styling

Reply:
Yes @jaimakena. You can select an element in Figma and in the panel in the rigth side, you can see the properties. But use those as a guide because not all the properties shown there will be useful for you. 

![HyF-webshop](https://user-images.githubusercontent.com/36211640/142079334-c7065c96-6240-4cdb-bf0b-9f1d56b517f6.jpg)

I add an image as an example.  I show in blue the element I selected and the properties that correspond to it. In your case, select the one you need and inspect it.